x-pack/plugins/observability_solution/apm/server/routes/default_api_types.ts

```diff
@@ -48,3 +50,31 @@ export const transactionDataSourceRt = t.type({
     t.literal(RollupInterval.None),
   ]),
 });
+
+const BoolQueryRt = t.type({
+  should: t.array(t.record(t.string, t.unknown)),
+  must: t.array(t.record(t.string, t.unknown)),
```

Comment:
Should we remove those two properties as we are not using them?

Reply:
Honestly, I'm trying to match the actual `BoolQuery` type from `kbn/es-query`, which is the return type of `buildFiltersFromQueries`, as close as possible.